src/client/push-client.js

```diff
@@ -111,7 +111,15 @@ export default class PushClient {
       scope: this.scope
     });
     await registrationReady(reg);
-    let sub = await reg.pushManager.subscribe({userVisibleOnly: true});
+    let sub = await reg.pushManager.subscribe({userVisibleOnly: true})
+      .catch((err) => {
+        // This is provide a more helpful message when work with Chrome + GCM
+        var errorToThrow = err;
```

Comment:
I'll give it a shot. Only problem I've had with that rule is let isn't
allowed outside of a scope which we may hit.

I'll add it to this PR if that's cool with you.

On Tue, 19 Jan 2016, 13:39 Mat Scales <notifications@github.com> wrote:

> In src/client/push-client.js
> <https://github.com/GoogleChrome/Propel/pull/14#discussion_r50112277>:
>
> > @@ -111,7 +111,15 @@ export default class PushClient {
> >        scope: this.scope
> >      });
> >      await registrationReady(reg);
> > -    let sub = await reg.pushManager.subscribe({userVisibleOnly: true});
> > +    let sub = await reg.pushManager.subscribe({userVisibleOnly: true})
> > +      .catch((err) => {
> > +        // This is provide a more helpful message when work with Chrome + GCM
> > +        var errorToThrow = err;
>
> We should add the no-var ESLint rule, actually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleChrome/Propel/pull/14/files#r50112277>.
>